modules/nf-cid/src/main/nextflow/data/cid/CidStore.groovy

```diff
@@ -49,12 +47,6 @@ interface CidStore {
      */
     Object load(String key)
 
-    /**
-     * Get the CID store location path.
-     * @return CID store location path.
-     */
-    Path getPath()
```

Comment:
It looks like this was used only for testing. Better to remove for the interface to keep more generic